### DISCUSSION TITLE
Extracting IfcWall verts but without openings

### DISCUSSION THREAD

**Discussion by jurvanoerle** [IfcOpenShell/IfcOpenShell#4460] [Q&A] [answered]
With the example code I managed to extract the verts of all walls from my ifc model (python), but the openings in the walls are included, so I get a lot of verts per wall if the wall includes openings. 
I would like to extract the verts of the wall but without openings (so only 8 verts for a simple wall with eg 3 openings)

regards, Jur

**Comment by Moult** [comment, marked as answer]
See https://blenderbim.org/docs-python/ifcopenshell-python/geometry_processing.html on how to calculate verts of any element.

See https://blenderbim.org/docs-python/ifcopenshell/geometry_settings.html#disable-opening-subtractions on how to disable any openings.
